src/main/webapp/EditHelp.js

```diff
@@ -0,0 +1,100 @@
+Behaviour.specify("A.help-button", 'hudson-behavior', 1, function(e) {
+        e.onclick = function() {
+            var tr = findFollowingTR(this, "help-area");            
+            var thistr = $(this).up().up();
+
+            if (!$(tr).hasClassName("custom-help")) {
+                var customtr = document.createElement("tr");
+                customtr.innerHTML = '  <td></td><td colspan="2"><div class="help"><div class="custom-hep-button">'+
+                '<div class="custom-hep-button-save" onclick="save_custom_help(this)" >Save</div><div class="custom-hep-button-edit" onclick="edit_custom_help(this)" >Edit</div>'+
+                '</div><div class="custom-hep-text">Loading1...</div></div></td><td></td>';
+                $(customtr).addClassName("help-area");
+                $(customtr).addClassName("custom-help");
+                $(thistr).insert({after:customtr});
+                $(customtr).down().next().down().down().down().next().style.display = "block";
```

Comment:
Very dependent on current DOM structure, therefore fragile. Can't this be solved better?